website/docs/cloud-docs/run/modes-and-options.mdx

```diff
@@ -17,6 +26,44 @@ Terraform Cloud runs support many of the same modes and options available in the
 - **API:** Use the `is-destroy` option.
 - **UI:** Use a workspace's **Destruction and Deletion** settings page.
 
+## Plan Only/Speculative Plan
+
+This option creates a [speculative plan](/terraform/cloud-docs/run/remote-operations#speculative-plans). The speculative plan shows a set of possible changes and checks them against Sentinel policies, but Terraform cannot apply it.
+
+You can create speculative plans with a Terraform version that is different from the one currently selected for the workspace. This lets you check whether your configuration is compatible with a newer Terraform version without changing the workspace settings.
+
+Plan-only runs ignore the per-workspace run queue; they can proceed even if another run is in progress, they cannot become the workspace's current run, and they do not block progress on other runs in the workspace.
+
+- **API:** Set the `plan-only` option to `true` and specify an available terraform version using the `terraform-version` field.
+- **UI:** Open the workspace's **Actions** menu, select **Start new run**, and then choose **Plan only** as the run type.
+- **VCS:** When a workspace is connected to a VCS repository, Terraform Cloud automatically starts a speculative plan when a pull request (or merge request) is opened against the selected branch of that repository. The plan will be linked from the pull/merge request view in your VCS, and can also be found in the workspace's run list.
+
+## Saved Plans
+
+-> **Version note:** Using saved plans from the CLI with Terraform Cloud requires at least Terraform CLI v1.6.0.
+
+Saved plan runs are very similar to standard plan and apply runs: they perform a plan, and then optionally apply it. There are three main differences:
+
+1. _No wait for planning._ Saved plan runs ignore the per-workspace run queue during their plan and checks; like plan-only runs, they can begin planning even if another run is in progress, without blocking progress on other runs. They only become the current run (and block other runs) once they are confirmed for apply.
```

Comment:
I think we can omit the last sentence because you cover that in the below point:
```suggestion
1. _No wait for planning._ Saved plan runs ignore the per-workspace run queue during their plan and checks. Like plan-only runs, saved plans can begin planning even if another run is in progress, without blocking progress on other runs.
```